ganga/GangaDirac/Lib/Backends/DiracBase.py

```diff
@@ -1160,91 +1160,99 @@ def finalise_jobs(allJobs, downloadSandbox = True):
 
         #First grab all the info from Dirac
         inputDict = {}
-        #I have to reduce the no. of subjobs per process to prevent timeouts
+        #I have to reduce the no. of subjobs per process to prevent DIRAC timeouts
         nPerProcess = int(math.floor(configDirac['maxSubjobsPerProcess']/2))
```

Comment:
Why the /2 here. Is that because it is the same number that control the number of jobs for submission. If yes, should it not be two different parameters instead?